lib/src/presentation/view/cards_list.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:flutter/material.dart';
+import'../../data/model/card_model.dart';
+import '../../data/repository/cards_repository_impl.dart';
+import '../../core/util/constants.dart';
+import 'card_detail.dart';
+
+class CardList extends StatelessWidget {
+  const CardList({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return SafeArea(
+      child: Scaffold(
+        body: FutureBuilder<List<CardModel>>(
+          future: getCards(),
+          builder: (BuildContext context,
+              AsyncSnapshot<List<CardModel>> snapshot,) {
```

Comment:
looks like this code need to be formatted

Reply:
Done